staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -339,7 +339,7 @@ func (t *tracker) getWatches(gvr schema.GroupVersionResource, ns string) []*watc
 		if w := t.watchers[gvr][ns]; w != nil {
```

Comment:
```suggestion
		if ns != metav1.NamespaceNone {
			if w := t.watchers[gvr][ns]; w != nil {
				watches = append(watches, w...)
			}
		}
		if w := t.watchers[gvr][metav1.NamespaceAll]; w != nil {
			watches = append(watches, w...)
		}
```

this would be cleaner